### ISSUE TITLE
My Stuff - search bar

### ISSUE THREAD

**Issue by jwzimmer** [LLK/scratch-www#1622]
Suggested in a workshop on 10/16/17: having a search bar in order to search your own projects/ studios in My Stuff.

We should consider this when we migrate My Stuff from scratchr2 to www, if there is re-design of the page at that time.

**Comment by GreenBayRules** [comment]
Ooh that's a nice idea. Especially for people with tons of projects.

**Comment by VutonDesign** [comment]
@GreenBayRules Like me lol

**Comment by kyleplo** [comment]
Yeah, I like that idea. Also, maybe you could filter by user in search:
`cats author:kyleplo` would return cat projects by @kyleplo

**Comment by vutondesign** [comment]
@kyleplo So, Similar to the Github issue and PR search feature?

**Comment by kyleplo** [comment]
@vutondesign Yes, see #1750

**Comment by amazinigmech2418** [comment]
This would be amazing! Now I won't have to do ctrl. + f for an hour to sort through my over 600 projects!

**Comment by kyleplo** [comment]
@amazinigmech2418 You have 600 projects?

**Comment by amazinigmech2418** [comment]
Yeah. over 200 are unshared.
